scripts/config/jest/babelTransform.js

```diff
@@ -0,0 +1,6 @@
+const babelJest = require('babel-jest');
```

Comment:
This shouldn't be in the `scripts` directory. I suggest putting it in `src/jest/babelTransfrom`